tests/test_browser.py

```diff
@@ -2217,8 +2216,6 @@ def test_openal_capture_sanity(self):
 
   def test_runtimelink(self):
     for wasm in [0, 1]:
-      if not wasm and self.is_wasm_backend():
-        continue
```

Comment:
This (and others) adds more test coverage, is that intentional? (previously `if not wasm` we would continue, but no longer)

It probably makes sense to do, but I'd need to read the test in more detail to have an opinion.

Reply:
Yes there are some places where it looks like we were not testing the WASM=0 path with wasm backend.  I think they probably just predated wasm2js being complete.  Where I found such a case and the tests passed I enabled the test.

Would you like me to split those out?

Reply:
No, no need to split it out. But I asked because it means I'll need to think about it as I review.